utils/getScenarioContent.tsx

```diff
@@ -1,75 +1,86 @@
 import { Claim, ClaimDetailsContent, ClaimStatusContent, ScenarioContent } from '../types/common'
 
-export type ScenarioTypeKey = keyof typeof ScenarioType
-
 export enum ScenarioType {
-  PendingDetermination = 'Pending determination scenario',
-  BasePending = 'Base state with pending weeks',
-  BaseNoPending = 'Base state with no pending weeks',
+  Scenario1,
+  Scenario7,
+  Scenario8,
+  Scenario9,
+  Scenario10,
+}
+
+export const ScenarioTypeNames = {
+  [ScenarioType.Scenario1]: 'Pending determination scenario',
+  [ScenarioType.Scenario7]: 'Base state; No pending weeks; No weeks to certify',
+  [ScenarioType.Scenario8]: 'Base state; No pending weeks; Has weeks to certify',
+  [ScenarioType.Scenario9]: 'Base state; Has pending weeks; No weeks to certify',
+  [ScenarioType.Scenario10]: 'Base state; Has pending weeks; Has weeks to certify',
 }
 
 /**
  * Identify the correct scenario to display.
  *
- * @param {Qbject} claim
- * @returns {Object}
+ * @TODO: Validating the API gateway response #150
  */
 export function getScenario(claimData: Claim): ScenarioType {
   // The pending determination scenario: if claimData contains any pendingDetermination
   // objects
   // @TODO: refactor with more detailed pending determination scenarios #252
   if (claimData.pendingDetermination && claimData.pendingDetermination.length > 0) {
-    return ScenarioType.PendingDetermination
+    return ScenarioType.Scenario1
   }
-  // The generic pending scenario: if there are no pendingDetermination objects
-  // AND hasPendingWeeks is true
-  else if (claimData.hasPendingWeeks === true) {
-    return ScenarioType.BasePending
-  }
-  // The generic "all clear"/base state scenario: if there are no pendingDetermination objects
-  // and hasPendingWeeks is false
-  else if (claimData.hasPendingWeeks === false) {
-    return ScenarioType.BaseNoPending
-  }
-  // This is unexpected
-  // @TODO: Log the scenario and display 500
+
+  // Otherwise display a Base State scenario.
   else {
-    // throw new Error('Unexpected scenario')
```

Comment:
Note: The new logic I went over with Nicole and Diana shouldn't leave an opportunity for an unknown scenario. We will also have an additional layer of API gateway response validation (#150) that will make unexpected response values not possible.